README.md

```diff
@@ -69,32 +69,46 @@ Borderlands 3 from EGS store.
 4. Now the launcher can search 'Catnip' and 'egs' as the codename and store in the database and correlate it with Borderlands 3 and ULWGL-12345. It can then feed ULWGL-12345 to the ulwgl-run script.
 
 
-# Building:
+# Building and packaging:
+Building ULWGL currently requires `bash`, `make` and `meson`
 
-TODO:
-Right now for the steam runtime we are just copying valve's builds. In the future this will be replaced with a proper full build to allow us to customize the runtime.
-
-Reaper:
-
-```
-cd reaper
-meson setup builddir
-cd builddir
-meson compile
+To build ULWGL, after downloading and extracting the source code from this repository, change into the newly extracted directory
+```shell
+cd ULWGL-Launcher
 ```
-Resulting 'reaper' executable will be inside builddir folder.  
 
-# Packaging:
-
-ULWGL entire folder should be placed at /usr/share/ULWGL  
-
-Copy ulwgl-run to /usr/bin/ulwgl-run  
+To configure the installation `PREFIX` (this is not related to wine's `WINEPREFIX`) use the `configure.sh` script
+```shell
+./configure.sh --prefix=/usr
+```
+Change the `--prefix` as fit for your distribution, for example `/usr/local`, or `/app` for packaging through flatpak
 
-For the flatpak version, the folders will be /app/share and /app/bin respectively.  
+Then run `make` to build. After a successful build the resulting files should be available in the `./builddir` directory
 
-TODO:  
+## Installing 
+To install ULWGL run the following command after completing the steps described above
+```shell
+make install
+```
+or if you are packaging ULWGL
+```shell
+make DESTDIR=<packaging_directory> install
+```
 
-We are working on a build script to allow custom INSTALL_PATH so that /usr/share and/or /app/share are not hard-coded required values. The build script will also provide a compiled copy of reaper.  
+## Installing as user
+If you want to install ULWGL just for your user, or for quickly testing, you can configure ULWGL with the following command
```

Comment:
In the future, we should probably move this section in a separate page since the primary audience is for developers/testers and distributors should only be running the Makefile to package this application

Reply:
This is still going through the Makefile, but indeed this is more relevant to testing that end users.